Source/IdentityApi/IdentityApi/Interfaces/IUserLocationManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿using IdentityApi.Models;
+
+namespace IdentityApi.Interfaces
+{
+    public interface IUserLocationManager
+    {
+        /// <summary>
+        /// Checks if ip adress is blacklisted 
+        /// </summary>
```

Comment:
Explain what is being returned, same for all the calls.

Ex: <Returns>True if address is blocked, false if not</returns>